srcpkgs/newsboat/template

```diff
@@ -16,11 +17,13 @@ license="MIT"
 homepage="https://newsboat.org/"
 changelog="https://raw.githubusercontent.com/newsboat/newsboat/master/CHANGELOG.md"
 distfiles="https://newsboat.org/releases/${version}/newsboat-${version}.tar.xz"
-checksum=0c46b3dd46bb578dd6dd4915db4cfdffb4352ab258f251080ad14655c75a9c31
+checksum=8920f41cc53776b67c0e85ad1696b0967f6ac248f3b8913d977942c15d75e690
 
-do_check() {
-	make test
-	(cd test && TERM=$TERM TMPDIR=/dev/shm ./test)
+# tests fail when run by superuser
+make_check=extended
+
+post_patch() {
+	vsed -i Makefile -e "s|\./test|TERM=$TERM TMPDIR=/dev/shm \./test|g"
```

Comment:
Why do we force `/dev/shm` instead of good old `/tmp`?

Reply:
I took that from the old template but I don't mind removing it.

Reply:
If it works without, I think that'd be best.

Reply:
JFYI: I recommend `/dev/shm` in the README because tests create a lot of temporary files, and putting them onto a tmpfs (of which `/dev/shm` is a common instance) greatly speeds things up. If you're tight on RAM or don't need the extra speed, or if `/tmp` is already a tmpfs, using `/tmp` is fine.